src/ndarray/ndarray.cc

```diff
@@ -746,20 +746,20 @@ std::shared_ptr<ngraph::runtime::Tensor> &NDArray::create_tensor(
       ptr_->tensor_view_->get_shape() !=
           ngraph_bridge::TShape_to_NShape(shape_)) {
     auto backend = ngraph_bridge::GetBackendFromContext(ctx());
+    void* dptr = storage_handle().dptr;
     CHECK(backend != nullptr);
```

Comment:
I think if we already have a macro around these code, we could use `ngraph_bridge::check()` here like the rest of the code instead of `CHECK()`. Although, I don't know why we created a our own check in the first place rather than using the mxnet version.

Reply:
!!! Will fix. To answer your question: 

Klockwork doesn’t like MXNet’s `CHECK` macro, complains about various memory violations. When I added the negative test cases and actually hit one of those checks, instead of throwing an error that GTEST could catch and validate, `CHECK` returned `SIGABRT` or `SIGSEGV`. Rather than trying to fix the rabbit hole of macros, I just wrote our own utility and replaced all of the instances we’re using in the bridge.